impl/config/config.go

```diff
@@ -71,7 +71,7 @@ func GetDefaultConfig() Config {
 			BaseURL:     "http://localhost:8305",
 			LogLocation: "log",
 			LogLevel:    "debug",
-			DBFile:      "diddht.db",
+			DBFile:      "bolt://diddht.db",
```

Comment:
this might be problematic since unix won't be able to create files of this format

Reply:
see PR description (updated at almost the exact time you left this message): this config option is parsed as a URI and the scheme is used to determine what storage backend to use. The default behavior is unchanged, it still uses bolt with a file called `diddht.db` in the current directory.

Reply:
ok cool!